src/wallet/rpcwallet.cpp

```diff
@@ -2183,9 +2185,17 @@ static RPCHelpMan lockunspent()
 
     bool fUnlock = request.params[0].get_bool();
 
+    bool store_lock = false;
+    if (!request.params[2].isNull()) {
+        RPCTypeCheckArgument(request.params[2], UniValue::VBOOL);
+        store_lock = request.params[2].get_bool();
+    }
```

Comment:
nit: Univalue get_* will already do type checking, so can drop `RPCTypeCheckArgument`.

```suggestion
    const bool store_lock{request.params[2].isNull() ? false : request.params[2].get_bool()};
```

Reply:
Good point!